eng/scripts/Create-APIView.ps1

```diff
@@ -322,4 +339,45 @@ function New-RestSpecsAPIViewReviews {
     Write-Host "##vso[task.complete result=SucceededWithIssues;]DONE"
     exit 1
   }
-}
\ No newline at end of file
+}
+
+<#
+.DESCRIPTION
+  Get all the tags from the swagger readme file with the default tag indicated.
+
+.PARAMETER ReadMeFilePath
+  The file path to the readme file.
+#>
+function Get-TagInformationFromReadMeFile {
+  param (
+    [Parameter(Mandatory = $true)]
+    [string]$ReadMeFilePath
+  )
+  $tags = [System.Collections.Generic.HashSet[string]]::new()
+  $markDownContent = Get-Content -Path $ReadMeFilePath
+  $checkForDefaultTag = $false
+  $defaultTag = $null
+
+  foreach ($line in $markDownContent) {
+    if ($line -match "### Basic Information") {
+      $checkForDefaultTag = $true
+    }
+
+    if ($checkForDefaulttag -and ($null -eq $defaultTag)) {
+      if ($line -match "^tag:\s*(?<tag>.+)") {
+        $defaultTag = $matches["tag"]
+        $checkForDefaultTag = $false
+      }
+    }
+
+    if ($line -match '^```\s*yaml\s*\$\(tag\)\s*==\s*''(?<tag>.+)''') {
+      $tag = $matches["tag"]
+      $tags.Add($tag) | Out-Null
```

Comment:
out of curiosity why do you need `Out-Null` here?

Reply:
It outputs True or False depending on if the command succeeded.